gulp/config.js

```diff
@@ -65,7 +65,7 @@ module.exports = {
         paths.modules + '/cf-pagination/src',
         paths.modules + '/cf-tables/src',
         paths.modules + '/cf-typography/src',
-        paths.modules + '/cfpb-chart-builder/src/static/css',
+        paths.modules + '/cfpb-chart-builder/src/**',
```

Comment:
Yeah, the old version of the chart library had its Less in `/cfpb-chart-builder/src/static/css`. The new version has it in `/cfpb-chart-builder/src/static/`. To make cfgov-refresh future-proof I added the `**` which will search all subdirectories inside of `src` because it's possible we'll change the Less' location down the road.

The capital framework component paths above *only* search in `src` and not `src`'s subdirectories.